src/participant/participant.ts

```diff
@@ -1096,11 +1118,7 @@ export default class ParticipantController {
       vscode.CancellationToken
     ]
   ): Promise<ChatResult> {
-    const [request, context, stream, token] = args;
-    const abortController = new AbortController();
-    token.onCancellationRequested(() => {
-      abortController.abort();
-    });
+    const [request, context, stream] = args;
```

Comment:
Why not decompose token here as well and pass it as a named variable further down rather than as `args[3]`?